data/autoyast_sle15/create_hdd/create_hdd_ha_aarch64.xml.ep

```diff
@@ -0,0 +1,326 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
This profile should support part of SLE15+ while not or SLE12+, so suggest to add 'sle15' at least in the names.

Reply:
I think we made a mistake here @lemon-suse there was another folder with sle12, so we didn't need the prefix, like in this case :)

Reply:
Yes, you are right, the folder is sle15. While in actual using for the profiles in the folder, we haven't distinguish much.(That is, even a case for SLE12+ we will use profile in the folder also.)